lib/tasks/job_board.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :job_board do
+  desc 'Send digest of current posted jobs to Slack'
+  task digest: :environment do
+    slack_client = SlackClient.new
+    jobs_channel_id = 'C030WQBRZFH' # TODO: Currently this is #jobs-test, update to the #jobs channel ID
```

Comment:
I don't thiiink I have access to see what the real channel ID should be here, so someone Slack privileges can feel free to make the change, or just let me know what to update the value to :)

Reply:
How would you feel about making this an ENV var, that way somebody like us could use a different channel ID to test it locally, but it would post to the proper channel on production?